pkg/kubectl/cmd/cmd_test.go

```diff
@@ -201,11 +201,17 @@ func NewAPIFactory() (*cmdutil.Factory, *testFactory, runtime.Codec) {
 		Validator: validation.NullSchema{},
 	}
 	generators := map[string]kubectl.Generator{
```

Comment:
service/test exists only for a test case I had added in order to verify that a generated resource that doesn't use a label selector, won't break oc expose. It will be removed after https://github.com/kubernetes/kubernetes/pull/17516 since that test isn't needed any more.